.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,64 @@
+on: [push, pull_request]
+
+name: Continuous-integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
```

Comment:
Can you please add `--all` to each of the cargo commands? We have some additional crates in the workspace that need to get checked as well